src/index.html

```diff
@@ -1,13 +1,51 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/styles.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+        src="./images/imac.jpeg" 
+        alt="logo" 
+        class="card__img"
+        >
+      <h2 class="card__tittle">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h2>
+      <p class="card__desctiption">
+        Код товара: 195434
+      </p>
+      <div class="card__rating-container">
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__feedback">
+          Отзывов: 5
+        </p>
+      </div>
+      <div class="card__price">
+        <h3 class="card__price-tittle">
+          Цена:
+        </h3>
+        <p class="card__price-value">
+          69 999 грн
+        </p>
+      </div>
+      <a href="#" class="card__button" data-qa="hover">
```

Comment:
Когда атрибутов больше 2, их надо разделять
```suggestion
      <a 
        href="#" 
        class="card__button" 
        data-qa="hover"
      >
```